front/src/views/recovery-password/RecoveryPassword.css

```diff
@@ -0,0 +1,35 @@
+.col {
+    flex-grow: 0;
+}
+
+.v-card__actions {
+    padding: inherit;
+}
+
+.theme--light.v-stepper {
+    background: #F6F8FB;
```

Comment:
i see this value again
please, add it into variables.css and use variable

Reply:
Ah, I think I understand